10.IDR_metadata/notebooks/run_preprocessing_and_aggregation.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I am guessing this is slurm processing instructions - it would be ideal to add a brief README within the `10.IDR_metadata` repo clarifying what this module does and how it does it.

Reply:
addressed